src/controllers/users.controller.js

```diff
@@ -0,0 +1,81 @@
+const { usersService } = require('./../services');
+
+const getAll = (req, res) => {
+  const users = usersService.getAll();
+
+  res.json(users);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const user = usersService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.json(user);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = usersService.create(name);
```

Comment:
It's a good practice to handle potential errors from the `usersService.create` method, such as database errors or duplicate entries, and respond with an appropriate status code and message.